VideoLocker/AndroidManifest.xml

```diff
@@ -225,7 +225,7 @@
         <activity android:name=".profiles.UserProfileActivity"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme.NoShadow"
-            android:label=""/>
+            android:label="@string/profile_title"/>
```

Comment:
(Alternatively, we could replace `""` in that code with "Profile"), if we still want the username to become the title when scrolling)